tests/suite/test_jwt_policies_jwksuri.py

```diff
@@ -41,7 +56,7 @@ def get_token(request):
 
 @pytest.mark.skip_for_nginx_oss
 @pytest.mark.policies
-@pytest.mark.skip(reason="issues with ingressClass")
+# @pytest.mark.skip(reason="issues with ingressClass")
```

Comment:
@shaun-nx can we remove this comment, probably not needed after fixing the count issue

Reply:
I'll remove it from the new VSR test as well